packages/devp2p/src/dpt/server.ts

```diff
@@ -149,20 +147,7 @@ export class Server extends EventEmitter {
     this.debug(typename, debugMsg)
 
     const msg = encode(typename, data, this._privateKey)
-    // Parity hack
-    // There is a bug in Parity up to at lease 1.8.10 not echoing the hash from
-    // discovery spec (hash: sha3(signature || packet-type || packet-data))
-    // but just hashing the RLP-encoded packet data (see discovery.rs, on_ping())
-    // 2018-02-28
-    if (typename === 'ping') {
-      const rkeyParity = keccak256(msg.slice(98)).toString('hex')
-      this._parityRequestMap.set(rkeyParity, msg.slice(0, 32).toString('hex'))
-      setTimeout(() => {
-        if (this._parityRequestMap.get(rkeyParity) !== undefined) {
-          this._parityRequestMap.delete(rkeyParity)
-        }
-      }, this._timeout)
-    }
```

Comment:
This is unrelated to the original problem, I just removed "on occasion", but not to confuse anyone.